CONTRIBUTING.md

```diff
@@ -175,6 +175,8 @@ any PMC member:
 1. Create a new commit updating the [spec.md](spec.md) and [profile](profile.md)
    headers to include the version of the release
    (`Open Service Broker API (v$major.$minor)`).
+1. Create a new commit updating the [oepnapi.yaml](openapi.yaml) version to be
```

Comment:
Not sure why you didn't just include the yml in the previous list of files to update, but ok...

Reply:
Yeah. I don't know why either. Updating!

Reply:
(also realised I spelt `openapi.yaml` wrong!